cime/config/e3sm/testmods_dirs/bench/fcomp_noio/shell_commands

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+./xmlchange HIST_OPTION=none
+./xmlchange SAVE_TIMING=TRUE
+# on KNLs, run hyper-threaded with 64x2
+if [ `./xmlquery --value MACH` == theta ]||[ `./xmlquery --value MACH` == cori-knl ]; then
+  ./xmlchange MAX_MPITASKS_PER_NODE=64
+  ./xmlchange MAX_TASKS_PER_NODE=128
+  ./xmlchange NTHRDS=2
+else
+  ./xmlchange NTHRDS=1
+fi
```

Comment:
Does "fcomp_noio" mean this is for modifying an F-case?  Then it should be in cam/cime_config/testmods